index.js

```diff
@@ -24,67 +25,84 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
-
+  let guess = parseInt(guessInput.value, 10);
+  console.log(guess)
+  
   hideAllMessages();
 
+
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+
+    
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
+    numberOfGuessesMessage.style.display ='';
```

Comment:
When I guess the right number, there's a javascript error in the console pointing to this line. It says that `numberOfGuessesMessage` is null. To figure out why that would be, I would suggest double checking the code where `numberOfGuessesMessage` is first declared in this file. Is the query selector function there looking for the correct class name to be able to get the element that we're looking for?